### ISSUE TITLE
Uncaught TypeError: Cannot read property 'isDestroyed' of null

### ISSUE THREAD

**Issue by sparkofreason** [mauricioszabo/atom-chlorine#156]
[Enter steps to reproduce:]

1. Occurred when evaluating an ns form.

Rolling back to 0.4.8 seems to fix it.

**Atom**: 1.44.0 x64
**Electron**: 4.2.7
**OS**: Mac OS X 10.15.3
**Thrown From**: [chlorine](https://github.com/mauricioszabo/atom-chlorine) package 0.4.9

### Stack Trace

Uncaught TypeError: Cannot read property 'isDestroyed' of null

```
At /Users/dave.dixon/.atom/packages/chlorine/lib/cmds.js:1510

TypeError: Cannot read property 'isDestroyed' of null
    at j1 (/packages/chlorine/lib/cmds.js:1510:241)
    at zaa (/packages/chlorine/lib/cmds.js:1512:111)
    at sca (/packages/chlorine/lib/cmds.js:1810:293)
    at H1 (/packages/chlorine/lib/cmds.js:1549:257)
    at Q4 (/packages/chlorine/lib/cmds.js:1784:1041)
    at /packages/chlorine/lib/cmds.js:1792:362
    at P4 (/packages/chlorine/lib/cmds.js:1784:818)
    at Function.e (/packages/chlorine/lib/cmds.js:1792:317)
    at Function.f (/packages/chlorine/lib/cmds.js:1792:442)
    at /packages/chlorine/lib/cmds.js:1821:251)
    at HTMLElement.<anonymous> (/Users/dave.dixon/.atom/init.coffee:69:14)
    at CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:349290)
    at KeymapManager.dispatchCommandEvent (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:1253613)
    at KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:1249747)
    at WindowEventHandler.handleDocumentKeyEvent (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:284496)
```

### Commands

```
     -1:05.3.0 lisp-paredit:expand-selection (atom-text-editor.editor.indent-mode-76f60.is-focused)
     -1:05.3.0 core:delete (atom-text-editor.editor.indent-mode-76f60.is-focused)
     -1:05.2.0 datatip:toggle (input.hidden-input)
     -1:04.9.0 core:backspace (input.hidden-input)
     -1:01.4.0 sparkofreason:inspect-s-exp (input.hidden-input)
     -1:01.4.0 lisp-paredit:expand-selection (atom-text-editor.editor.indent-mode-76f60.is-focused)
     -1:01.4.0 lisp-paredit:contract-selection (atom-text-editor.editor.indent-mode-76f60.is-focused)
     -0:59 chlorine:connect-clojure-socket-repl (input.hidden-input)
  2x -0:45.6.0 parinfer:next-tab-stop (input.hidden-input)
     -0:44.6.0 core:move-down (input.hidden-input)
 13x -0:44.4.0 core:move-left (input.hidden-input)
  2x -0:42.4.0 parinfer:next-tab-stop (input.hidden-input)
     -0:14.2.0 chlorine:connect-clojure-socket-repl (input.hidden-input)
     -0:11.4.0 chlorine:clear-console (input.hidden-input)
     -0:09.4.0 sparkofreason:inspect-s-exp (input.hidden-input)
     -0:09.4.0 lisp-paredit:expand-selection (atom-text-editor.editor.indent-mode-76f60.is-focused)
```

### Non-Core Packages

```
atom-ide-ui 0.13.0 
atom-java-decompiler 0.1.2 
busy-signal 2.0.1 
chlorine 0.4.9 
go-debug 1.7.0 
go-plus 6.1.0 
go-signature-statusbar 1.2.3 
ide-go 0.12.3 
ink 0.12.2 
intentions 1.1.5 
linter 2.3.1 
linter-kondo 1.2.0 
linter-ui-default 1.8.1 
lisp-paredit 0.6.0 
parinfer 1.23.0 
split-diff 1.6.1 
```

**Comment by mauricioszabo** [comment]
Ok, there are _a lot_ of problems while trying to support two different implementations of "inline results" that both respond to `.isDestroyed` but one is a method and the other is an attribute, and that _even then_ they can become `null` for whatever reason :laughing: 

I'll publish a new version soon, can you please test it to see if it's fixed? Thanks :)

**Comment by mauricioszabo** [comment]
Fixed on 0.4.10

**Comment by sparkofreason** [comment]
Looks good! Thanks you!
